docs/public/robots.txt

```diff
@@ -0,0 +1,2 @@
+User-agent: *
+Disallow:
```

Comment:
```suggestion
Allow: /

Sitemap: https://docs.dataform.co/sitemap.xml
```

Sitemap not required but does help; easiest option is to use an [externally generated sitemap](https://www.xml-sitemaps.com/details-docs.dataform.co-8ee834eda.html), but could add a package to do it instead.

`Disallow: /404.html` can be added when there's a 404 page.